graphql_compiler/tests/integration_tests/test_backends_integration.py

```diff
@@ -27,12 +30,36 @@
     test_backend.MYSQL,
     test_backend.SQLITE,
     test_backend.MSSQL,
+    test_backend.NEO4J,
+    test_backend.REDISGRAPH,
+])
+
+# TODO Leon this is really bad but it works for now
```

Comment:
Yeah let's not do this. Why can't we generate the parameterization with toggle-able backends? Ideally, we'd like to be able to do things like `@test_against_all_backends(except_redisgraph=True)` or something like that, instead of writing one of these for every combination of backends we'd like to include or exclude.

Reply:
Commit aedc4a9 does exactly that now :)